src/m3ninx/index/segment/fst/encoding/docs/slice.go

```diff
@@ -72,7 +68,7 @@ func (r *SliceReader) Doc(id postings.ID) (doc.Document, error) {
 }
 
 // AllDocs returns a docs iterator.
-func (r *SliceReader) AllDocs() index.IDDocIterator {
+func (r *SliceReader) Iter() index.IDDocIterator {
```

Comment:
Needs a comment change above.

Reply:
Can probably also just delete the comment since it's implementing the interface